katas/5kyu/Vector class #1/solution.py

```diff
@@ -21,11 +21,11 @@
 class Vector:
     def __init__(self, *args):
         self.vector = list(args[0]) if len(args) == 1 else list(args)
-        self.x, self.y, self.z = self.vector
+        self.x, self.y, *self.z = self.vector
```

Comment:
tests would fail here as `*self.z` is an array here

Reply:
done